stdlib/http/src/main/java/org/ballerinalang/net/http/websocket/client/InitEndpoint.java

```diff
@@ -137,6 +130,105 @@ private static void populateClientConnectorConfig(MapValue<String, Object> clien
         return customHeaders;
     }
 
+    /**
+     * Populate the retry config.
+     *
+     * @param retryConfig - the retry config.
+     * @param retryConnectorConfig - the retry connector config.
+     */
+    private static void populateRetryConnectorConfig(MapValue<String, Object> retryConfig,
+                                                     RetryContext retryConnectorConfig) {
+        setInterval(Integer.parseInt(retryConfig.get("intervalInMillis").toString()), retryConnectorConfig);
```

Comment:
Why don't we use retryConfig.getIntValue()? Then we can do some proper handling like in 
https://github.com/ballerina-platform/ballerina-lang/blob/eb2f6f10668efe06eef50698d98d4ad371dc8d5e/stdlib/http/src/main/java/org/ballerinalang/net/http/websocket/WebSocketUtil.java#L147-L159

Shall we fix in all following places where integer is parsed?